FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/mqtt/mqtt_plain_text/DemoTasks/PlaintextMQTTExample.c

```diff
@@ -297,7 +297,7 @@ static void prvMQTTDemoTask( void * pvParameters )
     NetworkContext_t xNetworkContext = { 0 };
     MQTTContext_t xMQTTContext;
     MQTTStatus_t xMQTTStatus;
-    BaseType_t xNetworkStatus;
+    PlaintextTransportStatus_t xNetworkStatus;
```

Comment:
Should we instead consider updating the demo to directly call the `FreeRTOS_sockets_wrapper.h` function for connect/disconnect instead of using 2 layers of calls from `Plaintext_FreeRTOS_Connect` -> `Sockets_Connect` -> FreeRTOS+TCP functions

Reply:
There is a task for considering that as an option. It will be decided as part of that. I have talked to @abhidixi11 about this for the scope of this PR, NeworkContext is to be maintained. We will reconsider it when we get to the task.

Reply:
Okay, I would have preferred that we made the correction in this PR itself instead of having the 2 layers of indirection to FreeRTOS+TCP. However, will not hold this PR though!

Reply:
Ok Thanks.